mlflow/transformers.py

```diff
@@ -1795,6 +1804,31 @@ def _parse_lists_of_dict_to_list_of_str(self, output_data, target_dict_key) -> L
         else:
             return output_data[target_dict_key]
 
+    @staticmethod
+    def _parse_feature_extraction_input(input_data):
+        if isinstance(input_data, list) and isinstance(input_data[0], dict):
+            return [list(data.values())[0] for data in input_data]
+        else:
+            return input_data
+
+    @staticmethod
+    def _parse_feature_extraction_output(output_data):
+        """
+        Parse the return type from a FeatureExtractionPipeline output.
+        Examples:
+
+        Input: [[[0.11, 0.98, 0.76]]] or np.ndarray(0.11, 0.98, 0.76)
```

Comment:
Is `np.ndarray(0.11, 0.98, 0.76)` valid?

```
>>> np.ndarray(0.11, 0.98, 0.76)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: expected a sequence of integers or a single integer, got '0.11'
>>> np.ndarray([0.11, 0.98, 0.76])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'float' object cannot be interpreted as an integer
>>> np.array([0.11, 0.98, 0.76])
array([0.11, 0.98, 0.76])
```

Reply:
ahhhh good catch. The doc string is missing the proper representation. the input from Tensor parsing within model serving will be np.ndarray([0.11, 0.98, 0.76]). updating!